src/app/_components/Card.tsx

```diff
@@ -59,13 +66,16 @@ export function Card({
       )}
     >
       {image?.url && (
-        <div className="relative block h-52">
+        <div className="relative block aspect-video h-52">
```

Comment:
is the h-52 needed? Btw responsiveness looks really nice, super smooth

Reply:
Yes. Needed.

WITH
<img width="1035" alt="Screenshot 2024-05-02 at 15 03 58" src="https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/26620750/724b19c7-523f-4836-91df-eda44c25d609">

WITHOUT
<img width="1072" alt="Screenshot 2024-05-02 at 15 04 05" src="https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/26620750/9d5c2ba9-089d-4b70-a324-2cd26ad7499c">